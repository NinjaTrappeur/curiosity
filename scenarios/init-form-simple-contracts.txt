as mila
forms simple-contract new --amount "-100"
forms simple-contract new
forms simple-contract new --amount 100
