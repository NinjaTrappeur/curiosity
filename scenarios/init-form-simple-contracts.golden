1: as mila
Modifying default user.
2: forms simple-contract new --amount -100
UserNotFound "mila"
3: forms simple-contract new
UserNotFound "mila"
4: forms simple-contract new --amount 100
UserNotFound "mila"
